.travis.yml

```diff
@@ -2,10 +2,11 @@ language: php
 php:
   - 7.4
   - 7.3
+  - 7.2
 sudo: false
 dist: trusty
 install:
   - composer install --dev
 script: vendor/phpunit/phpunit/phpunit tests
 matrix:
-  allow_failures:
\ No newline at end of file
+  allow_failures:
```

Comment:
Should we remove this `matrix` section since it doesn't seem to be getting used?